proto/backend.proto

```diff
@@ -58,13 +73,12 @@ message StringList {
 }
 
 message CallResourceRequest {
-  PluginConfig config = 1;
-  User user = 2;
-  string path = 3;
-  string method = 4;
-  string url = 5;
-  map<string,StringList> headers = 6;
-  bytes body = 7;
+  PluginContext context = 1;
```

Comment:
```suggestion
  PluginContext pluginContext = 1;
```

given how reservered the word `context` is in go I think we can be more explicit about the naming.